apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -0,0 +1,32 @@
+import { html } from 'lit';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { PieButton } from '@justeattakeaway/pie-button';
+
+const componentRef = [PieButton];
+
+const meta: Meta = {
+    title: 'pie-button',
+    component: 'pie-button',
+};
```

Comment:
### Added component reference in story

Storybook 7 uses CSF 3 for writing stories, and now the field `component` uses a string instead of the component reference. 

Since the component is not referenced anywhere, it doesn't get initialized, therefore it doesn't render.

To circumvent the issue I added a constant to hold it's reference.

`const componentRef = [PieButton];`

Unfortunately I didn't found a canonical way to solve this, only found another user with a [similar issue](https://discord.com/channels/486522875931656193/490822506916347904/1052209727146307614).